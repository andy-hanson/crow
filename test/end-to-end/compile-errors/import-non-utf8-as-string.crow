import
	./not-utf8.txt as s string

main void()
	info log s
