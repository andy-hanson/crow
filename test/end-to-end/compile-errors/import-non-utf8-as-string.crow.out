exit code: 1
stderr:
test/end-to-end/compile-errors/import-non-utf8-as-string.crow 2:5-2:31 File is not encoded as UTF-8 or has encoding errors.
Stopping due to compile errors.
